runtimes/browser.js

```diff
@@ -20,6 +20,12 @@ var $ = window.$ = {
     var fwin = frame.contentWindow;
     var fdoc = fwin.document;
     var fscript = fdoc.createElement('script');
+
+    // The following is a workaround for a bug in Chromium related to reporting
+    // errors produced from evaluating code using `eval`.
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=746564
+    fdoc.write('<body>');
```

Comment:
:D Believe it or not, it breaks Edge. Edge is currently not working anyway, though, so I will address that later.